### ISSUE TITLE
React

### ISSUE THREAD

**Issue by oshri1997** [malte-wessel/react-textfit#70]
In react 17.0.1v doesnt work

**Comment by iway1** [comment]
Yes it does. Make sure that your `<Textfit>` isn't inside of a flex container.

**Comment by petersaints** [comment]
Duplicate of #67. I also made a suggestion of a work around on that other issue: https://github.com/malte-wessel/react-textfit/issues/67#issuecomment-786184206
